hal/lib/athena/Notifier.cpp

```diff
@@ -134,8 +134,8 @@ HAL_NotifierHandle HAL_InitializeNotifier(void (*process)(uint64_t, HAL_Notifier
   std::shared_ptr<Notifier> notifier = std::make_shared<Notifier>();
   HAL_NotifierHandle handle = notifierHandles.Allocate(notifier);
   if (handle == HAL_kInvalidHandle) {
-	  *status = HAL_HANDLE_ERROR;
-	  return HAL_kInvalidHandle;
+         *status = HAL_HANDLE_ERROR;
+         return HAL_kInvalidHandle;
```

Comment:
Yea. cpplint fixed the indentation, then I undid it manually and didn't realize it was a tab.